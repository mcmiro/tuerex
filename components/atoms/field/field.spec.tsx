/* eslint-disable no-use-before-define */
import * as React from "react";
import {screen, render} from "@testing-library/react";
import Field from "./";

describe("Test Field component", () => {
	it("Should pass when name, variant, size and placeholder is additional", () => {
		render(<Field data-testid="test" />);

		const renderFiledComponent = screen.queryAllByTestId("test");
		expect(renderFiledComponent).not.toBeNull();
		expect(renderFiledComponent).toBeDefined();
		expect(renderFiledComponent).toMatchSnapshot();
	});
});
