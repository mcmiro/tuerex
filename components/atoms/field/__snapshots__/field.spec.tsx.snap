// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Field component Should pass when name, variant, size and placeholder is additional 1`] = `[]`;
